content/applications/hr/payroll/payslips.rst

```diff
@@ -0,0 +1,402 @@
+
+Lastly, in the :guilabel:`Company` field, select the company these payslips are written against.
+
+When all the information on the batch payslip detail form is correct, click the :guilabel:`Save`
+button to save the information. To delete the entry, click the :guilabel:`Discard` button.
+
+To create the payslips for the newly-created batch, click the :guilabel:`Generate Payslips` button
+at the top of the form.
+
+When clicked, a pop-up window appears showing all the payslips that will be created. To remove any
+individual payslips, click the black :guilabel:`🗙 (clear)` icon at the far right of the payslip
+line.
+
+If a specific :guilabel:`Salary Structure` or :guilabel:`Department` needs to be specified for the
+batch, select them from the corresponding drop-down menus.
+
+Click the :guilabel:`Generate` button at the bottom of the pop-up window to create the payslips for
+the batch.
+
+.. image:: payslips/generate-payslips.png
+   :align: center
+   :alt: Generata payslips for the new batch.
+
+.. note::
+   On the :guilabel:`Generate Payslips` pop-up window, selecting a :guilabel:`Department` and/or
+   :guilabel:`Salary Structure` only displays payslips that apply to those specifically-selected
+   parameters.
+
+If there are any errors or issues preventing the payslips from being generated, a detailed error
+message appears in the top-right section. This error box disappears on its own after several
+seconds, or the 'X' can be clicked to close the alert.
+
+To remedy the issue, make any necessary modifications (e.g. removing any payslip lines that cannot
+be processed), then click the :guilabel:`Generate` button again.
+
+Once the payslips have been successfully generated, the screen returns to the batch payslip detail
+form.
+
+From here, click the :guilabel:`Generate Draft Entry` button to change the payslips status from
+*Draft* to *Done*.
+
+Once payslips have been generated, click the :guilabel:`Make Payment` button to process the
+payments. Doing so reveals a pop-up window, in which the proper banking information must be entered.
+In this pop-up window, select the appropriate :guilabel:`Bank Journal` from the drop-down menu, and
+enter the appropriate file name.
+
+When done, click the :guilabel:`Confirm` button to confirm the information, or click
+:guilabel:`Cancel` to discard.
+
+.. image:: payslips/confirm-payslips.png
+   :align: center
+   :alt: Confirm the bank details for the new payslips.
+
+Generate commission payslips
+----------------------------
+
+Commission payslips can be generated directly from the :guilabel:`Payslips Batches` page
+(:menuselection:`Payroll app --> Payslips --> Batches`). To generate commission payslips from this
+page, click on the desired batch (or batches) to create commissions payslips for, then click the
+:guilabel:`Generate Commission Payslips` button.
+
+.. image:: payslips/commission.png
+   :align: center
+   :alt: Click the Generate Commission Payslips to create commission payslips.
```

Comment:
This one too– can be removed. These are suggestions to reduce the amount of images in the doc, they certainly are helpful so please keep them if you feel like they should stay 🙂
```suggestion
```

Reply:
I trust your judgement! I removed these- I realized they can seem redundant. I tend to add too many pics, so I appreciate the feedback.